### DISCUSSION TITLE
Can you share the SRS of the project?

### DISCUSSION THREAD

**Discussion by imranertaza** [edrisranjbar/qurandl#28] [Q&A] [answered]
Hi
Can you please share the SRS of the project so that contributors can understand about the requirements and how this project is going to be? We could develop things gradually according to the SRS.
Thank you

**Comment by edrisranjbar** [comment]
we do not have a `SRS` yet, if you can help it would be grate. we can have a discussion around this topic together.
@AbdollahAshrafi can help us regarding this matter.

**Comment by imranertaza** [comment]
Hi
As you know everything about the project of how you want this project will be and what are the features you need into this project, you have to write it down in any document of PDF file. As you are the project owner, you should know about the features you need. You also also write an another general discussion writing all those features so that contributors can understand this and can contribute on this project.
Thank you

**Comment by edrisranjbar** [comment]
will be done ASAP

**Comment by edrisranjbar** [comment, marked as answer]
initial .SRS.md file added to project root directory. how should we proceed from here? any idea? shall we add wiki? or we can create a directory for Documentation and put this file over there.
